build_package.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
can you move this and the other script into a subdirectory so it's not polluting the top-level... maybe util/?

Reply:
moved to gutil